R/hail_install.R

```diff
@@ -0,0 +1,45 @@
+#' Install Hail and Datasets
+#' 
+#' Install hail dependencies and datasets to run the examples in documentation.
+#' To remove hail use \code{hail_uninstall}. 
+#'   
+#' @param datasets_examples If TRUE, hail will be downloaded along with the datasets to
+#' run the examples. Use FALSE if you just want to install hail.
+#' 
+#' @examples 
+#' \dontrun{
+#'  hail_install()
+#'  hail_uninstall()
+#' }
+#' @name hail_install
+NULL
+
+#' @rdname hail_install
+#' @export
+hail_install <- function(datasets_examples = TRUE){
+  hail_exists <- system.file("java/hail-all-spark.jar", package = "sparkhail")
+  
+  if (hail_exists != ""){
+    message("Hail is already installed.")
+  } else {
+    hail_all_spark <- "https://raw.githubusercontent.com/samuelmacedo83/sparkhail/master/inst/java/hail-all-spark.jar"
```

Comment:
Rather than downloading from `master`, I think we should move this to `r-spark` and then create a branch for this first release, say, `0.1.0` or whichever version you decide. I would then push that branch and install from that branch. This to avoid breaking previous versions of the package when you update `hail.all.spark.jar` in the future.